pkg/modelzooserver/input_check.go

```diff
@@ -13,12 +13,39 @@
 
 package modelzooserver
 
+import (
+	"fmt"
+	"regexp"
+)
+
 // checkName checks model definition and trained model is valid.
-func checkName(name string) bool {
-	return true
+func checkName(name string) error {
```

Comment:
How about adding a few unit tests as samples?

Reply:
Thanks, I just forgot to `git add` the test file...